CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# CONTRIBUTING GUIDELINES
+
+NovelRT is a community-driven project. Contribution is always welcome, and much appreciated.
+It is also how it'll be developed.
+
+First, please take a moment to review our [code of conduct](CODE_OF_CONDUCT.md). We expect that all project participants will adhere to it, and any actions taken that go against the Code of Conduct will not be tolerated.
+
+These guidelines are an attempt at better addressing the incoming issues and pull requests. 
+Please read them closely, and feel free to ask any questions if needed.
+
+## Getting started
+### Submitting Pull Requests
+
+You might know the basics of 
+[contributing on GitHub](https://help.github.com/articles/using-pull-requests) and have a fork
+[properly set up](https://github.com/ohmyzsh/ohmyzsh/wiki/Contribution-Technical-Practices).
+
+You can only create PRs on a dedicated branch.
+
+If you create your own PR, please double check that everything is all right. Also please type what you fixed or added in the PR body,
+[for instance](https://help.github.com/articles/closing-issues-via-commit-messages/)
+
+Your PR should be directed to the ```main``` branch - but should start from another fork of this repo.
+If you need help / information - please ask for help! Especially if you're a ```first-timer```. 
+
+
+#### If you contribute to the development, remember to keep this in mind.
+[How you can build NovelRT](README.md) :)
+We use a code formatter named ```clang-format```, so our codestyle is similar to Microsoft's. 
+If you find it hard to adhere to the code style, our CI process will produce a patch for you when neededm so that you can apply it and make your code adherent.
```

Comment:
Fixing a spelling error and tweaking the wording a little with this suggestion.

```suggestion
If you find it hard to adhere to the code style, our CI process will produce a patch for you when needed so that you can apply it and make your code consistent with the rest of the project.
```